jdaviz/cli.py

```diff
@@ -16,14 +16,11 @@
 
 @click.version_option(__version__)
 @click.command()
+@click.argument('layout',
+                nargs=1,
+                type=click.Choice(['cubeviz', 'specviz', 'mosviz', 'imviz'],
+                                  case_sensitive=False))
 @click.argument('filename', nargs=1)
-@click.option('--layout',
-              default='default',
```

Comment:
Are we deleting the "default" config? Seems like it is not possible to be used from CLI with this patch.

Reply:
Yes, @camipacifici suggested removing it entirely until it gets proper support/documentation (if we ever get around to that) to prevent users from finding it in its half-baked state.

Reply:
But we still let people do it from `Application(...)` call?

Reply:
right, should we remove that as well?  I'm guessing that's not as easy to stumble-upon though?

Reply:
Perhaps not, but we are carrying around dead code, so what's the point?